themes/default/layouts/what-is/single.html

```diff
@@ -1,68 +1,79 @@
-                            </div>
-                            <div>
-                                <div class="accordion-item text-2xl py-3 border-b-2 border-t-2">
-                                    {{ partial "accordian-header" (dict "text" "Transcript" "large_header" false) }}
+    <div class="container mx-auto">
+        <div class="lg:flex lg:justify-end">
+            <div class="lg:max-w-5xl">
+                {{ with .Params.video }}
+                <div class="card p-6 my-12">
+                    <h4>{{ .title }}</h4>
+                    <div>
+                        <div class="" style="position: relative; padding-bottom: 48.25%; height: 0; overflow: hidden;">
+                            <iframe src="{{ .youtube_url }}"
+                                style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;"
+                                allowfullscreen=""></iframe>
+                        </div>
+                        <p>
+                            {{ .description }}
+                        </p>
+                    </div>
+                    <div>
+                        <div class="accordion-item text-2xl py-3 border-b-2 border-t-2">
+                            {{ partial "accordian-header" (dict "text" "Transcript" "large_header" false) }}
 
 
-                                    <div class="accordion-item-body-no-animation text-base">
-                                        <p>{{ .transcript | markdownify }}</p>
-                                    </div>
-                                </div>
+                            <div class="accordion-item-body-no-animation text-base">
+                                <p>{{ .transcript | markdownify }}</p>
                             </div>
                         </div>
-                    {{ end }}
+                    </div>
+                </div>
+                {{ end }}
 
 
-                    <div class="bg-white px-3">
-                        {{ .Content }}
-                    </div>
+                <div class="bg-white px-3">
+                    {{ .Content }}
                 </div>
             </div>
         </div>
-    </section>
+    </div>
+</section>
 
-    <section id="get-started" class="container px-6 lg:px-0 mx-auto my-16">
-        <div class="w-full bg-violet-600 card p-6 lg:p-16 lg:pt-24 text-center">
-            <div class="max-w-xl mx-auto">
+<section id="get-started" class="container px-6 lg:px-0 mx-auto my-16">
+    <div class="w-full bg-violet-600 card p-6 lg:p-16 lg:pt-24 text-center">
+        <div class="max-w-xl mx-auto">
+            {{ if strings.HasPrefix .Params.page_title "Run " }}
+                <h2 class="text-white hidden lg:block px-0 lg:px-16">Get started with Pulumi ESC today</h2>
+                <h4 class="text-white mt-0 lg:hidden">Get started today</h4>
+                <p class="text-white">Pulumi ESC is open source and free to get started.</p>
```

Comment:
hey does this change actually affect ALL what-is pages? I think we just need these command pages to direct to ESC. the other ones should follow the general Pulumi IaC get started.